spec/facter/facts/aix/networking/interfaces_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+describe Facts::Aix::Networking::Interfaces do
+  describe '#call_the_resolver' do
+    subject(:fact) { Facts::Aix::Networking::Interfaces.new }
+
+    let(:value) do
+      {
+        'ens160' => {
+          'bindings' => [
+            {
+              'address' => '10.16.116.8',
+              'netmask' => '255.255.240.0',
+              'network' => '10.16.112.0'
+            }
+          ]
+        }
+      }
+    end
+
+    before do
+      allow(Facter::Resolvers::Aix::Networking).to receive(:resolve).with(:interfaces).and_return(value)
+    end
+
+    it 'calls Facter::Resolvers::NetworkingLinux' do
+      fact.call_the_resolver
+      expect(Facter::Resolvers::Aix::Networking).to have_received(:resolve).with(:interfaces)
+    end
+
+    it 'returns networking.interfaces fact' do
```

Comment:
Could you add also the test for when the resolver returns nil. This applies to all the fact tests.

Reply:
Done